.github/workflows/upgrade-python-requirements.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       # optional parameters below; fill in if you'd like github or email notifications
       # user_reviewers: ""
       # team_reviewers: ""
-      email_address: "masters-requirements-update@2u-internal.opsgenie.net"
+      email_address: "aurora-requirements-update@2u-internal.opsgenie.net"
```

Comment:
yes, this bone's connected to the (screencap) bone
<img width="801" alt="Screen Shot 2022-07-28 at 3 28 53 PM" src="https://user-images.githubusercontent.com/6405097/181621642-f706d020-0402-4e32-9086-b6d3802cd711.png">